src/main/java/eggventory/Eggventory.java

```diff
@@ -30,6 +33,20 @@ public static void main(String[] args) {
         storage = new Storage(filePath);
         parser = new Parser();
         stockList = storage.load();
+        //loanList = new LoanList();
+
+        /*
+        Calendar date = Calendar.getInstance();
+        loanList.addLoan("R500", "A123", 100, date, date);
+        loanList.addLoan("R500", "A6000", 100, date, date);
+        loanList.addLoan("ARDUINO", "A123", 100, date, date);
+        loanList.addLoan("NO", "A12", 100, date, date);
+
+        System.out.print("All: \n" + loanList.printLoans());
+        System.out.print("A123: \n" + loanList.printPersonLoans("A123"));
+        System.out.print("R500: \n" + loanList.printStockLoans("R500"));
+        */
+
```

Comment:
Do you want to remove your commented out lines of code?

Reply:
right right thanks